content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -690,7 +690,7 @@ These situations are:
 
 ### Setting the field validation level
 
-  {{< feature-state for_k8s_version="v1.25" state="beta" >}}
+  {{< feature-state for_k8s_version="v1.27" state="stable" >}}
 
 Provided that the `ServerSideFieldValidation` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled (disabled
```

Comment:
It's good to revise this section, and review the whole document, to frame things differently: validation is server-side, and client-side validation is now deprecated (right?).

_Optionally_, a client can request either strict validation, or no validation.

Elsewhere in the docs, we should document the warnings that you get from `kubectl` when the input manifest (or partial configuration) either duplicates a field, or refers to a field that the API server doesn't recognise. The key point to bear in mind when revising is the new origin of the warnings. However, the final docs should be [timeless](https://developers.google.com/style/timeless-documentation) and describe the current approach, rather than the change to the new thing.

**Aside**: we ought to have done that wider review when this moved to beta; for graduation to GA, it's essential - there's no later point in the KEP lifecycle to pick up those changes.

If there are big changes needed to docs that another SIG would lead on maintaining, it's OK to instead file _and freeze_ an issue that tracks the changes we need.

Finally, please fix the typo in
```markdown
server-side field validation when sending requests to a serer with this feature
```

Reply:
I've reworded the introduction and removed references to the "enableability" of the feature-gate throughout (given that server-side validation is now locked to enabled in GA).

Hopefully this reads more timeless, let me know if anything is off.

I've also opened and froze an issue to track the need to update the docs once client side validation is fully removed from kubectl.